draft-ietf-httpbis-priority.md

```diff
@@ -510,7 +512,7 @@ type PROTOCOL_ERROR.
 ~~~ drawing
 HTTP/2 PRIORITY_UPDATE Frame {
   Length (24),
-  Type (8) = 0x10,
+  Type (i) = 10,
```

Comment:
Note that this reverts another PR you've previously taken.  Probably a bad conflict resolution.

Reply:
Nice catch! FIxed in 834f062.